### DISCUSSION TITLE
Can't find the dataset for training the model

### DISCUSSION THREAD

**Discussion by A4dii** [larics/icuas24_competition#21] [General]
This is regarding Benchmark 1. 
For identifying the Models of the plants with fruits, where is the dataset from which we can train our model for detection. We do not have any dataset of pictures which have the plants with the fruits. In the models folder in the icuas24_competition, the only picture files available are the basic plant texture, but we do not have anything for the different sorts of fruits. 
How are we supposed to get the dataset to train our models.

**Comment by fpetric** [comment]
Hi @A4dii, 
for the simulation phase, the fruits are of different color and simple image processing techniques should be enough to detect the fruits. If your really need a dataset, you can create your own using .dae files that contain the 3D models of each fruit.
